packages/react-composites/src/composites/CallWithChatComposite/components/MoreDrawer.tsx

```diff
@@ -107,8 +107,9 @@ export const MoreDrawer = (props: MoreDrawerProps): JSX.Element => {
         // But this is a known issue in our state.
         onSelectSpeaker(selected as AudioDeviceInfo);
       }
+      props.onLightDismiss();
     },
-    [speakers, onSelectSpeaker]
+    [speakers, onSelectSpeaker, props]
```

Comment:
avoid `props` as this means any prop change will cause this to be recreated.
Instead update the above `const { speakers, onSelectSpeaker } = props;`
to have the onLightDismiss:
`const { speakers, onSelectSpeaker, onLightDismiss } = props;`
and put `onLightDismiss` into the dependency array

Reply:
^^ I second this as well!

Reply:
updated :)